easybuild/easyblocks/m/metis.py

```diff
@@ -123,9 +123,22 @@ def sanity_check_step(self):
         if LooseVersion(self.version) < LooseVersion("5"):
             dirs += ["Lib"]
 
+        libext = []
+        if isinstance(self.cfg['configopts'], list):
+            for elements in self.cfg['configopts']:
+                if 'shared=1' in elements:
```

Comment:
OK, thanks for clarifying that.

But, we should still move the check which libraries should get installed to `configure_step`, to avoid dealing with the list/string status of `configopts`.

I'll send a PR your way for that.